WordPress/Classes/ViewRelated/Blog/BlogListViewController.m

```diff
@@ -563,6 +569,9 @@ - (void)setEditing:(BOOL)editing animated:(BOOL)animated
     if (editing) {
         [self updateHeaderSize];
         self.tableView.tableHeaderView = self.headerView;
+
+        self.firstDisable = 0;
```

Comment:
That's my bad, I was originally storing an `NSTimeInterval` and refactored it at the last minute without doing a proper check.

I don't know whether it's a good idea to store the date here as it consumes part of the timeout interval, from the moment "Edit" is pressed instead of the first disable. If this change is made perhaps the interval needs to be increased slightly?